easy_rec/python/layers/input_layer.py

```diff
@@ -243,7 +250,8 @@ def single_call_input_layer(self,
         features,
         group_columns,
         cols_to_output_tensors=cols_to_output_tensors,
-        feature_name_to_output_tensors=feature_name_to_output_tensors)
+        feature_name_to_output_tensors=feature_name_to_output_tensors,
+        is_training=self._is_training)
```

Comment:
use is_training for?

Reply:
for kv embedding_loookup, when is_training is false, return zero embedding vectors, and does not create a new entry in kv map